### ISSUE TITLE
Autoloading error

### ISSUE THREAD

**Issue by jhlms** [baoagency/polaris_view_components#284]
@kirillplatonov 

In some projects I get an error from the autoloaders in the engine config. Specifically it can't find the helpers. 

Changing `config.autoload_paths` to `config.autoload_once_paths` seems to fix the issue but not sure if this will have unintended consequences?

Also noted here #200

Thanks

**Comment by kirillplatonov** [comment]
Actually, I changed it from `autoload_once_paths` to `autoload_paths` a while ago: https://github.com/baoagency/polaris_view_components/commit/ee451b808967a16a4caa28be63aefc26d06a67dd
I guess the reason was that `autoload_once_paths` won't reload classes.

Do you have any tips on how to reproduce the problem? I suspect that it might be related to Zeitwerk 🤔

@resistorsoftware you had a similar issue in #200 which you managed to fix, maybe you have some ideas on how to can reproduce it?

**Comment by zhongsheng** [comment]
I have this problem when I put a line of code in config/initializers

Ruby 3.2.0
Rails 7.0.4.3

`rails new testapp`

```
# testapp/config/initializers/mail.rb

ActionMailer::Base.delivery_method = :smtp
```

@kirillplatonov

**Comment by kirillplatonov** [comment]
@zhongsheng thanks for the reproduction steps. Resolved in v0.13.3.

**Comment by zhongsheng** [comment]
@kirillplatonov I was playing with Rails 7.1 recently, this fix will cause loading error. Maybe remove this fix when Rails7.1 release in the future.

**Comment by kirillplatonov** [comment]
@zhongsheng I will test it out on 7.1. Thanks for the heads up!

**Comment by pedrofs** [comment]
I removed the `require`s inside `initializer "polaris_view_components.helpers"` from the `engine.rb`, and it worked. Not sure what is going on.

**Comment by kirillplatonov** [comment]
@pedrofs thanks for the tip! The fix is coming in `v1.4`

**Comment by hecbuma-snap** [comment]
👋  for some reason I keep getting the autoloading error, I'm migrating from 7.0 to 7.1.X my solution is to requiere the engine in the Gemfile

```ruby
gem "polaris_view_components", "~> 2.1.0", require: "view_component/engine
```
